src/localizations/en.ftl

```diff
@@ -106,61 +106,78 @@ register-ticket-day-validation-errors-day-required = Please select a day.
 register-ticket-level-title = Select your ticket
 
 register-ticket-level-card-standard =
-  This Standard Convention ticket grants you access to the convention from Wednesday to Sunday.
+  This Standard Convention ticket grants you access to the convention.
 
   + Standard convention badge
   + program book
-  + participation in all scheduled events
+  + participation in scheduled events
 
   .label = Standard
   .priceLabel = Standard ticket
+  .priceLabelDay = Standard day ticket
```

Comment:
Instead of creating extra properties for this we can just do
```
.priceLabel = {$type ->
 *[full] Standard ticket
  [day] Standard day ticket
}
```